uMenu/source/ui/ui_Actions.cpp

```diff
@@ -75,38 +79,61 @@ namespace ui::actions {
     }
 
     void ShowControllerSupport() {
+        ui::quickmenu_utils::quickMenuInputsToIgnore=1;
         HidLaControllerSupportArg arg = {};
         hidLaCreateControllerSupportArg(&arg);
         arg.enable_explain_text = true;
         for(u32 i = 0; i < 8; i++) {
-            strcpy(arg.explain_text[i], "Test explain text");
+            strcpy(arg.explain_text[i], ""); //Controller explain test here
```

Comment:
We could make a string for this in the translation JSON actually, maybe myself or in a new PR

Reply:
In the original switch menu there is not any text if i remember correctly, maybe themes can make advantage of this is some ways but it will be a new functionality